src/zoo.js

```diff
@@ -11,57 +11,128 @@ eslint no-unused-vars: [
 
 const data = require('./data');
 
-function animalsByIds(ids) {
-  // seu código aqui
+function animalsByIds(...ids) {
+  if (!ids) return [];
+  return ids.map(id => data.animals.find(animal => animal.id === id));
```

Comment:
Fala meu querido! Só mostrando uma outra forma de fazer, que o cacique sugeriu no plantão:
```suggestion
  return data.animals.filter(animal => ids.includes(animal.id));
```
Dessa forma você só uma HoF (.filter) ao invés de usar duas (.map .find) e pode ganhar mais performance.

Reply:
Gostei... ficou mais fácil de ler!